### DISCUSSION TITLE
cant find the installer after download and unzip

### DISCUSSION THREAD

**Discussion by mrrcomp** [openssl/openssl#24374] [Q&A] [answered]
I downloaded the zip file and unzipped but cant find the installer??
Appreciate any help 
Regards
mrrcomp

**Comment by nhorman** [comment]
Can you elaborate on which installer you are referring to?  If you're asking about the zip file in our releases area, thats the source code, not any sort of installer executable.

**Reply by mrrcomp** [reply]
The Win64OpenSSL-<version>.exe ???

**Reply by nhorman** [reply]
That's not something that we currently produce.  Did you download it from shining light productions perhaps?  You'll have to contact them for support.

**Comment by mrrcomp** [comment]
No from the get hub and also from the ooenssl org website
________________________________

**Comment by nhorman** [comment]
You mean you  obtained your download from here:
https://openssl.org/source/index.html

Or here:
https://github.com/openssl/openssl/releases

As noted above, none of those files contain installers.  They contain the source code for openssl, which you must then compile to use.  If you're looking for a windows installer, we're developing one now, and hope to have an official release in the 3.5 time frame.

**Comment by mrrcomp** [comment]
Thanks its interesting every search 🔍 on install openssl gives the link 🔗 to the openssl. Org says download and ron the installer

Thanks

________________________________

**Reply by nhorman** [reply]
That Is interesting,  searching google for "install openssl" returns as it's first result an ai generated set of Instructions (next to a warning that says its experimental).  The instructions make a number of assumptions how software is installed in the generic sense, and they are definitely wrong for openssl (and ostensibly lots of other software).
What's worse, the next several results are for various companies that might have their clients make use of openssl, and they clearly just duplicated the ai results from google.

**Comment by mattcaswell** [comment, marked as answer]
There are a number of 3rd parties that produce OpenSSL installers. We maintain a set of links to them here:

https://wiki.openssl.org/index.php/Binaries
